templates/flamegraph.mustache

```diff
@@ -73,16 +75,22 @@ function init() {
         draw();
     }
 
-    window.excimerChart.label(function(d) {
-        return d.data.name + ' (' + ((d.x1 - d.x0) * 100).toFixed(3)
-                + '%, ' + d.value + ' samples, duration ~' +  d.value * {{samplerate}} + ' ms)'
-    });
+    displaysamplestats = function (d) {
```

Comment:
According to the issue, it should only display the decimal when things are in the realm of seconds.

I'm currently seeing this locally, and the conditions would need to be added here.

![image](https://user-images.githubusercontent.com/9924643/192429332-5fa6a024-8184-4e11-8ac2-d29346ca5b56.png)

Reply:
done